meta/event_counting.yml

```
@@ -0,0 +1,31 @@
+---
+# VMs
+vmware.vmware.guest_info:
+  query: >-
+    .guests[] | {
+      name: (.moid + "-" + .instance_uuid),
+      canonical_facts: {
+        ansible_vmware_instance_uuid: .instance_uuid,
+        ansible_vmware_bios_uuid: .hw_product_uuid,
+        ansible_vmware_moid: .moid
+      },
+      facts: {
+        device_type: "VM"
+      }
+    }
+
+vm_list_group_by_clusters_info:
```

```suggestion
vmware.vmware.vm_list_group_by_clusters_info:
```

I think this is obviously more consistent with the other entry.

---

yes, good catch :)